evap/static/js/.gitignore

```diff
@@ -2,3 +2,4 @@
 *.js
 !*.min.js
 !sortable_form.js
+!moment_de.js
```

Comment:
Unless there' value in these two entries here (which I currently don't see), I'd consider this duplication and thus rather have these removed
```suggestion
```